docs/ENGINE_FUNCTIONS.md

```diff
@@ -0,0 +1,274 @@
+| mission_checks()            | `mission_data`, `player`, `which_key`                                                                                                                                                                                                                                                                  | Return a boolean value, to tell if a mission condition `which_key` is completed, from its data `mission_data`                   |
+| execute_triggers()          | `mission_data`, `player`, `which_key`, `dialog`, `preferences`, `text_replacements_generic`, `drinks`, `item`, `enemy`, `npcs`, `start_player`, `lists`, `zone`, `mission`, `mounts`, `start_time`, `map`, `save_file`, `player_damage_coefficient`                                                    | Executes a mission trigger `which_key` from its data `mission_data`                                                              |
+| offer_mission()             | `mission_id`, `player`, `missions_data`, `dialog`, `preferences`, `text_replacements_generic`, `drinks`, `item`, `enemy`, `npcs`, `start_player`, `lists`, `zone`, `mission`, `mounts`, `start_time`, `map`, `save_file`, `player_damage_coefficient`, `enemies_damage_coefficient`, `previous_player` | Initiates the mission offering UI of mission id `mission_id`                                                                     |
+| mission_completing_checks() | `mission_id`, `missions_data`, `player`, `dialog`, `preferences`, `text_replacements_generic`, `drinks`, `item`, `enemy`, `npcs`, `start_player`, `lists`, `zone`, `mission`, `mounts`, `start_time`, `save_file`, `player_damage_coefficient`                                                         | Checks if mission id `mission_id` can be completed. If it can, mark it as completed and run its completing triggers if there are |
+
+## npc_handling.py
+
+The [`npc_handling.py`](https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/blob/master/source/npc_handling.py) class handles the interaction between the player and the npcs:
+
+| Name       | Arguments                                                                | Description                                                                                                      |
+|------------|--------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
+| init_npc() | `map_location`, `player`, `npcs`, `drinks`, `item`, `preferences`, `map` |  Inits the interaction loop between the player and the npc at the player's current map point digit `map_location` |
+
+## player_handling.py
+
+The [`player_handling.py`](https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/blob/master/source/player_handling.py) class handles all major actions that can be done on the player, as death:
```

Comment:
```suggestion
The [`player_handling.py`](https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/blob/master/source/player_handling.py) class handles all major actions that can be done on the player, such as death:
```
```suggestion
The [`player_handling.py`](https://github.com/Dungeons-of-Kathallion/Bane-Of-Wargs/blob/master/source/player_handling.py) class handles all major actions that can be done on the player, as in death:
```
Not sure which because there is only one function documented here.